src/pages/Interferometry/MachZehnderCanvas.js

```diff
@@ -2,21 +2,98 @@ import React, { useRef } from "react";
 import PropTypes from "prop-types";
 import useAnimationFrame from "../../useAnimation";
 import MachZehnderEquipment from "./MachZehnderEquipment";
+import Photon from "./Photon";
 
 const equipment = new MachZehnderEquipment(
   100,
   100,
-  400,
+  800,
   400,
 );
 
-function MachZehnderCanvas({ size: { width, height } }) {
+const generateNumberOfPhotons = (number) => {
```

Comment:
`generatePhotons` is fine.

Reply:
`number` is confusing to the `Number` primitive function.
`numPhotons` or `n` is better.

Reply:
I changed to numPhotons !